ui/static/js/app.js

```diff
@@ -293,6 +293,11 @@ class UnreadCounterHandler {
             let oldValue = parseInt(element.textContent, 10);
             element.innerHTML = callback(oldValue);
         });
+        // The following 3 lines ensure that the unread count in the titlebar
+        // is updated correctly when users presses "v".
+        let oldValue = parseInt(document.title.split('(')[1], 10);
+        let newValue = callback(oldValue);
+        document.title = 'Unread Items (' + newValue + ') - Miniflux';
```

Comment:
There is a problem here. You should not hard code the title in English because the application internationalized. In addition to that, if you are on another "items list" page which is not the unread page, the wrong title get replaced.